src/mongo/dbtests/clienttests.cpp

```diff
@@ -28,6 +28,8 @@
  *    then also delete it in the license file.
```

Comment:
@benety please review this. I dropped extra changes and made storage mock a singleton